### PR TITLE
Add Decentraland icon

### LINKED ISSUE #7141: Request: Decentraland
### Brand Name

Dentraland

### Website

https://decentraland.org/

### Popularity Metric

Alexa rank: top 2.5k global https://www.alexa.com/siteinfo/decentraland.org
website: https://decentraland.org
github: https://github.com/decentraland 
(over 4k+ github stars)

### Official Resources for Icon and Color

https://github.com/decentraland/ui/blob/master/static/logo.svg
color

### Additional Comments

*Yes. Decentraland and Decentraland MANA

*It is the worlds leading blockchain virtual world. 

### Contributing

No

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
Decentraland

**Issue:** closes #7141
**Popularity metric:** Similarweb rank [97514](https://www.similarweb.com/website/decentraland.org)

### Checklist
- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
Colour is from their [website](https://decentraland.org), referred to as `--primary` in the CSS variables.

### CODE REVIEW

File: icons/decentraland.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Decentraland</title><path d="M16.8883 18.9778h3.1112L16.8883 15.2ZM8.444 9.5556v6.6222h5.4667ZM12.0439 0C5.4661 0 .0883 5.3333.0883 11.9556c0 1.4666.2667 2.8888.7556 4.1777l4.6222-5.5555 2.6667-3.2445 6.4889 7.8223 1.8666-2.2667 4.889 5.8667h.4c1.3777-1.9556 2.1333-4.3112 2.1333-6.8445C23.9995 5.3333 18.6216 0 12.0439 0ZM8.0883 6.2667c-.9333 0-1.6444-.7556-1.6444-1.6445s.7556-1.6444 1.6444-1.6444c.9334 0 1.6445.7555 1.6445 1.6444 0 .889-.7111 1.6445-1.6445 1.6445zm8.6667 5.3777a3.2815 3.2815 0 0 1-3.2889-3.2888 3.2815 3.2815 0 0 1 3.289-3.289 3.2815 3.2815 0 0 1 3.2888 3.289c.0444 1.8222-1.4667 3.2888-3.2889 3.2888zm-3.9555 5.7334H1.2439c.2667.5333.5778 1.0222.8889 1.511h9.3778Zm-8.2667 4.0444h14.8889c.4889-.4 1.0222-.8889 1.3333-1.2444H3.1995c.4444.4444.8888.8889 1.3333 1.2444zM12.0439 24c1.9556 0 3.8222-.4444 5.4667-1.2889H6.5772C8.2217 23.5556 10.044 24 12.044 24Z"/></svg>
```
Comment (service-paradis) [inline icons/decentraland.svg:1]: According to https://decentraland.org/brand:

> Subject to these terms, you are only allowed to use the logo design that is available at the following link https://www.figma.com/file/pBiYuNALsSEFncGy3x6dIZ/DecentralandLogo?node-id=3162%3A117.

Note that the Figma link should be https://www.figma.com/file/L9CZTTRnkvBd4nQK8xiD85/Decentraland-Logo?type=design&node-id=3137-1051&mode=design&t=H8BCuBilLfwHcd5N-0 according to https://github.com/decentraland/catalyst/issues/1726.

To be compliant with their terms, I think we should use the logo in their Figma as source. When comparing your version to the "Filled, Isotype, Small, Light" version, I got some differences.

Also, just to be sure, was the precision of 4 necessary for this one?

**Review by service-paradis — CHANGES_REQUESTED** [review]
Thank you @PeterShaggyNoble, I left a few comments about the terms and conditions!

**Comment by PeterShaggyNoble** [comment]
Nice find on the GH issue, @service-paradis 👍🏻  I've used that as the `source` for now. The version I had used as a source was just squashed horizontally but I started from scratch using the same version as you did from the Figma file to be safe. A precision of 4 is, unfortunately, necessary here; I was getting a small shift in the circles at 3.

**Review by service-paradis — APPROVED** [review]
> Nice find on the GH issue, @service-paradis 👍🏻 I've used that as the source for now. The version I had used as a source was just squashed horizontally but I started from scratch using the same version as you did from the Figma file to be safe. A precision of 4 is, unfortunately, necessary here; I was getting a small shift in the circles at 3.

Great! In the next review of the letter D, we could take a look if they updated the link! The icon is now a perfect match!

**Pull request merged**